### DISCUSSION TITLE
Adding website source ?

### DISCUSSION THREAD

**Discussion by KevVal** [sherlock-project/sherlock#953] [Q&A] [answered]
Hi, i would know if possible to modify the website list ? I mean adding like Discord, and other social ? TeamSpeak or other ?

**Comment by SethFalco** [comment, marked as answer]
I don't think things like Discord or Team Speak are possible. 🤔 
Maybe their forums, but certainly not the running instances themselves.

Regardless, for open-source projects there's often a CONTRIBUTING.md file that explains how you can contribute to the project if you want to!

https://github.com/sherlock-project/sherlock/blob/master/CONTRIBUTING.md
